samples/simple-android/project.properties

```diff
@@ -0,0 +1 @@
+target=android-21
```

Comment:
In the `build.sbt` you have `platformTarget in Android := "android-22"`, why here is 21?